interceptor/interceptor.go

```diff
@@ -286,3 +410,36 @@ func fetch(this js.Value, args []js.Value) interface{} {
 	promise := promiseConstructor.New(js.FuncOf(promise_logic))
 	return promise
 }
+
+func getStatic(this js.Value, args []js.Value) interface{} {
+	url := args[0].String()
+
+	return js.Global().Get("Promise").New(js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		resolve := args[0]
```

Comment:
```suggestion
		resolve := args[0]
		reject := args[1]
```

Reply:
I didn't define the reject because it was not used